source/app/documentmanager.h

```diff
@@ -71,7 +72,12 @@ class DocumentManager
 
     bool hasOpenDocuments() const;
     void setCurrentDocumentIndex(int index);
+
     int getCurrentDocumentIndex() const;
+	size_t getDocumentListSize() const;
+	
+	/// Returns -1 if the file at filepath is not open, else it returns the index at which the already open file is at
+	int containsDocument(const std::string& filepath);
```

Comment:
Changed in commit fae1f35. Regarding the indentation issue it must be a github thing as my editor/options shows tabs as 4 spaces and it is aligned with the rest of the code in visual studio.